### PR TITLE
[WIP] Select correct GitHub clientIds in Navbar based on environment

### PR DESCRIPTION
**Pull request by dhanush027**
Added Staging and Production links on Navbar.js file using window location.

### CODE REVIEW

File: js/navbar.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
 const hamburger = document.querySelector('.hamburger');
 const navMenu = document.querySelector('.nav-menu');
+const checkURL = window.location.href;
+const stagingURL = "https://staging-welcome.realdevsquad.com/";
+const productionURL = "https://welcome.realdevsquad.com/";
+
+function checkWindowURL() {
+  if (checkURL == stagingURL) {
+    document.getElementById("user-login").href = "https://github.com/login/oauth/authorize?client_id=c4a84431feaf604e89d1";
+  } else if (checkURL == productionURL) {
```
Comment (akshay1502) [inline js/navbar.js:10]: `else if` and `else` doing the same, let's have only else block.

File: index.html [hunk 2]
```diff
@@ -95,7 +95,9 @@
         <div class="nav-login">
           <a
             class="btn-login"
-            href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+            id="user-login"
+            onclick="checkWindowURL()"
```
Comment (akshay1502) [inline index.html:99]: Let's not wait until the user clicks on the button. We want the link to be there, whether the user click it or not.

File: js/navbar.js [hunk 3]
```diff
@@ -1,5 +1,18 @@
 const hamburger = document.querySelector('.hamburger');
 const navMenu = document.querySelector('.nav-menu');
+const checkURL = window.location.href;
+const stagingURL = "https://staging-welcome.realdevsquad.com/";
+const productionURL = "https://welcome.realdevsquad.com/";
+
+function checkWindowURL() {
```
Comment (akshay1502) [inline js/navbar.js:7]: Could you use [DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) event here. And put the code on top of the file.

File: index.html (hunk 2 above)
Comment (dhanush027) [reply index.html:99]: Will do these changes

File: js/navbar.js [hunk 4]
```diff
@@ -1,3 +1,17 @@
+window.addEventListener('DOMContentLoaded',(event)=>{
+  const checkURL = window.location.href;
+  const stagingURL = "https://staging-welcome.realdevsquad.com/";
+  const productionURL = "https://welcome.realdevsquad.com/";
+
+  if (checkURL == stagingURL) {
+    document.getElementById("user-login").href = "https://github.com/login/oauth/authorize?client_id=c4a84431feaf604e89d1";
```
Comment (akshay1502) [inline js/navbar.js:7]: use a separate const variable for referencing `user-login`.

File: js/navbar.js [hunk 5]
```diff
@@ -1,3 +1,17 @@
+window.addEventListener('DOMContentLoaded',(event)=>{
+  const checkURL = window.location.href;
+  const stagingURL = "https://staging-welcome.realdevsquad.com/";
+  const productionURL = "https://welcome.realdevsquad.com/";
+
+  if (checkURL == stagingURL) {
```
Comment (akshay1502) [inline js/navbar.js:6]: use === here.

Comment (akshay1502) [inline js/navbar.js:6]: Let's check for production URL here, by that we will have stagingLoginURL for all other URLs.

Comment (dhanush027) [reply js/navbar.js:6]: Will change it.

File: js/navbar.js (hunk 4 above)
Comment (dhanush027) [reply js/navbar.js:7]: But here, user-login is just the id for anchor tag.
Do i have to this: const login="user-login";
document.getElementById(login).href = "o auth link";

**Review by akshay1502 — APPROVED** [review]

File: js/navbar.js [hunk 6]
```diff
@@ -1,3 +1,16 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+  const checkURL = window.location.href;
+  const productionURL = "https://welcome.realdevsquad.com/";
+  let userLogin = document.getElementById('user-login');
+
+  if (checkURL === productionURL) {
```
Comment (ankushdharkar) [inline js/navbar.js:6]: Suggestion:
```js
  const clientId = (checkURL === productionURL ? '23c78f66ab7964e5ef97' : 'c4a84431feaf604e89d1';
  userLoginEl.href = `https://github.com/login/oauth/authorize?client_id=${clientId}`;
```

Comment (akshay1502) [reply js/navbar.js:6]: Thanks @ankushdharkar . @dhanush027  please make the required changes.

**Pull request closed without merging**